nav2_dwb_controller/dwb_plugins/test/twist_gen.cpp

```diff
@@ -370,10 +366,10 @@ TEST(TrajectoryGenerator, twisty)
 
 TEST(TrajectoryGenerator, sim_time)
 {
-  auto nh = makeTestNode("sim_time");
+  auto nh = makeTestNode("sim_time",
+    rclcpp::Parameter("dwb.sim_time", sim_time),
+    rclcpp::Parameter("dwb.linear_granularity", 0.5)});
   const double sim_time = 2.5;
```

Comment:
This needs to be before make node, since the variable is not yet defined.

Reply:
Thank you! Moved `sim_time` creation up in 928af55